### DISCUSSION TITLE
create a training mode

### DISCUSSION THREAD

**Discussion by samsab1** [iNavFlight/inav#9705] [Ideas]
new to quad piloting, I only fly in direct view. horizon mode allows for quick learning while acro mode quickly puts me in difficulty. Is it possible to create a "training" mode which would allow you to have the best of the 2 modes, namely:
over an inclination range configurable in 0 (horizontal) and x° a “horizon” type behavior
beyond x° of inclination, “acro mode” type behavior
the value x being configurable as desired by potentiometer for example

the idea being to be able to master the acro mode better and better by starting with x =45° then gradually reducing this value of x to 0°

**Comment by MrD-RC** [comment]
Horizon is Angle until the sticks reach a threshold. The. It becomes Acro.
